containers/elabctl/elabctl.sh

```
@@ -0,0 +1,687 @@
+
+function uninstall
+{
+    stop
+
+    local -r backtitle="eLabFTW uninstall"
+    local title="Uninstall"
+
+    set +e
+
+    dialog --backtitle "$backtitle" --title "$title" --yesno "\nWarning! You are about to delete everything related to eLabFTW on this computer!\n\nThere is no 'go back' button. Are you sure you want to do this?\n" 0 0
+    if [ $? != 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)